pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -436,11 +435,19 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 			// https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md
 			// Rates are in bits per second, burst values are in bits.
 			bandwidthParam.IngressRate = int(ingress.Value())
-			bandwidthParam.IngressBurst = math.MaxInt32 // no limit
+			if ingressBurst != nil {
```

Comment:
I think this should be part of `ExtractPodBandwidthResources` otherwise the behaviour for `kubenet` and `cni` plugins will differ?

Reply:
kubenet  doesn't need set ingressBurst/egressBurst, kubenet and bandwidth plugin is different, I didn't use kubenet.

Reply:
I change `ExtractPodBandwidthResources` to `ExtractPodBandwidthParam`, and make sure behaviour for kubenet and cni plugins are same.